docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md

```diff
@@ -0,0 +1,321 @@
+      1. Select **Developer** from the top menu
+      2. Click on **Extrinsics** from the dropdown menu
+
+        ![Extrinsics Section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-1.webp)
+
+2. Choose the **AssetConversion** pallet and click on the **add_liquidity** extrinsic
+      1. Select the **AssetConversion** pallet
+      2. Choose the **addLiquidity** extrinsic from the list of available extrinsics
+
+        ![Add Liquidity Extrinsic](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-5.webp)
+
+3. Fill in the required fields:
+      1. **asset1** - the Multilocation of the first asset in the pool. In this case, it is the DOT token, which the following Multilocation represents:
+         ```javascript
+         {
+            parents: 0,
+            interior: Here
+         }
+         ```
+      2. **asset2** - the second asset's Multilocation within the pool. This refers to the PPM token, which the following Multilocation identifies:  
+         ```javascript
+         {
+            parents: 0,
+            interior: {
+              X2: [{PalletInstance: 50}, {GeneralIndex: 1112}]
+            }
+         }
+         ``` 
+      3. **amount1Desired** - the amount of the first asset that will be contributed to the pool
+      4. **amount2Desired** - the quantity of the second asset intended for pool contribution
+      5. **amount1Min** - the minimum amount of the first asset that will be contributed
+      6. **amount2Min** - the lowest acceptable quantity of the second asset for contribution
+      7. **mintTo** - the account to which the liquidity tokens will be minted
+      8. Click on **Submit Transaction** to add liquidity to the pool
+
+        ![Add Liquidity Fields](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-6.webp)
+
+        !!! warning
+            Ensure that the amounts of tokens provided have been minted previously and are available in your account before adding liquidity to the pool.
+
+        In this case, the liquidity provided to the pool is between DOT tokens and PPM tokens with the asset ID 1112 on Polkadot Asset Hub. The intention is to provide liquidity for 1 DOT token (u128 value of 1000000000000 as it has 10 decimals) and 1 PPM token (u128 value of 1000000000000 as it also has 10 decimals).
+
+After signing and submitting the transaction, the liquidity will be added to the pool. To verify the liquidity addition, check the **Explorer** section on the Polkadot.Js App interface and ensure that the **LiquidityAdded** event has been emitted.
+
+![Liquidity Added Event](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-7.webp)
+
+## Swapping Assets
+
+### Swapping From an Exact Amount of Tokens
```

Comment:
The two images have different extrinsics:
one has **swapTokensForExactTokens** and the other has **swapExactTokensForTokens**